### ISSUE TITLE
t/op/closure.t: blocker to strict-by-default

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#237]
As of commit ac6b7ccf73 in the alpha-dev-02-strict branch on Aug 18 2020, the file cited is failing.  Here is output from `cd t; ./perl harness -v <file>; cd -`:
```
# t/op/closure.t

# Lots of output

Global symbol "$global_scalar" requires explicit package name (did you forget to declare "my $global_scalar"?) at - line 15.
Global symbol "@global_array" requires explicit package name (did you forget to declare "my @global_array"?) at - line 16.
Global symbol "%global_hash" requires explicit package name (did you forget to declare "my %global_hash"?) at - line 17.
Global symbol "$global_scalar" requires explicit package name (did you forget to declare "my $global_scalar"?) at - line 27.
Global symbol "@global_array" requires explicit package name (did you forget to declare "my @global_array"?) at - line 28.
Global symbol "%global_hash" requires explicit package name (did you forget to declare "my %global_hash"?) at - line 29.
Execution of - aborted due to compilation errors.
# Failed test 59 - STDERR is silent at op/closure.t line 466
#      got "Global symbol \"$global_scalar\" requires explicit package name (did you forget to declare \"my $global_scalar\"?) at - line 15.\nGlobal symbol \"@global_array\" requires explicit package name (did you forget to declare \"my @global_array\"?) at - line 16.\nGlobal symbol \"%global_hash\" requires explicit package name (did you forget to declare \"my %global_hash\"?) at - line 17.\nGlobal symbol \"$global_scalar\" requires explicit package name (did you forget to declare \"my $global_scalar\"?) at - line 27.\nGlobal symbol \"@global_array\" requires explicit package name (did you forget to declare \"my @global_array\"?) at - line 28.\nGlobal symbol \"%global_hash\" requires explicit package name (did you forget to declare \"my %global_hash\"?) at - line 29.\nExecution of - aborted due to compilation errors.\n"
# expected ""
# Failed test 60 - exited cleanly at op/closure.t line 473
#      got "65280"
# expected "0"
# Error code 65280 = 0xFF00
Global symbol "$global_scalar" requires explicit package name (did you forget to declare "my $global_scalar"?) at - line 15.
Global symbol "@global_array" requires explicit package name (did you forget to declare "my @global_array"?) at - line 16.
Global symbol "%global_hash" requires explicit package name (did you forget to declare "my %global_hash"?) at - line 17.
Global symbol "$global_scalar" requires explicit package name (did you forget to declare "my $global_scalar"?) at - line 24.
Global symbol "@global_array" requires explicit package name (did you forget to declare "my @global_array"?) at - line 25.
Global symbol "%global_hash" requires explicit package name (did you forget to declare "my %global_hash"?) at - line 26.
Execution of - aborted due to compilation errors.
# Failed test 68 - STDERR is silent at op/closure.t line 466
#      got "Global symbol \"$global_scalar\" requires explicit package name (did you forget to declare \"my $global_scalar\"?) at - line 15.\nGlobal symbol \"@global_array\" requires explicit package name (did you forget to declare \"my @global_array\"?) at - line 16.\nGlobal symbol \"%global_hash\" requires explicit package name (did you forget to declare \"my %global_hash\"?) at - line 17.\nGlobal symbol \"$global_scalar\" requires explicit package name (did you forget to declare \"my $global_scalar\"?) at - line 24.\nGlobal symbol \"@global_array\" requires explicit package name (did you forget to declare \"my @global_array\"?) at - line 25.\nGlobal symbol \"%global_hash\" requires explicit package name (did you forget to declare \"my %global_hash\"?) at - line 26.\nExecution of - aborted due to compilation errors.\n"
# expected ""
# Failed test 69 - exited cleanly at op/closure.t line 473
#      got "65280"
# expected "0"
# Error code 65280 = 0xFF00

# etc., etc.,

op/closure.t .. 
ok 1
ok 2
...
ok 58 - All good
not ok 59 - STDERR is silent
not ok 60 - exited cleanly
ok 67 - All good
not ok 68 - STDERR is silent
not ok 69 - exited cleanly
ok 76 - All good
not ok 77 - STDERR is silent
not ok 78 - exited cleanly
ok 89 - All good
not ok 90 - STDERR is silent
not ok 91 - exited cleanly
ok 101 - All good
not ok 102 - STDERR is silent
not ok 103 - exited cleanly
ok 113 - All good
not ok 114 - STDERR is silent
not ok 115 - exited cleanly
ok 126 - All good
not ok 127 - STDERR is silent
not ok 128 - exited cleanly
ok 138 - All good
not ok 139 - STDERR is silent
not ok 140 - exited cleanly
ok 150 - All good
not ok 151 - STDERR is silent
not ok 152 - exited cleanly
ok 160 - All good
not ok 161 - STDERR is silent
not ok 162 - exited cleanly
ok 169 - All good
not ok 170 - STDERR is silent
not ok 171 - exited cleanly
ok 178 - All good
not ok 179 - STDERR is silent
not ok 180 - exited cleanly
ok 191 - All good
not ok 192 - STDERR is silent
not ok 193 - exited cleanly
ok 203 - All good
not ok 204 - STDERR is silent
not ok 205 - exited cleanly
ok 215 - All good
not ok 216 - STDERR is silent
not ok 217 - exited cleanly
ok 228 - All good
not ok 229 - STDERR is silent
not ok 230 - exited cleanly
ok 240 - All good
not ok 241 - STDERR is silent
not ok 242 - exited cleanly
ok 252 - All good
not ok 253 - STDERR is silent
not ok 254 - exited cleanly
ok 255 - RT \#9535
ok 256 - RT \#9535
ok 257 - RT \#9535
...
ok 282 - taking a ref to unavailable var should not copy it
1..282
Failed 186/282 subtests 

Test Summary Report
-------------------
op/closure.t (Wstat: 0 Tests: 132 Failed: 36)
  Failed tests:  59-60, 68-69, 77-78, 90-91, 102-103, 114-115
                127-128, 139-140, 151-152, 161-162, 170-171
                179-180, 192-193, 204-205, 216-217, 229-230
                241-242, 253-254
  Parse errors: Tests out of sequence.  Found (58) but expected (51)
                Tests out of sequence.  Found (59) but expected (52)
                Tests out of sequence.  Found (60) but expected (53)
                Tests out of sequence.  Found (67) but expected (54)
                Tests out of sequence.  Found (68) but expected (55)
Displayed the first 5 of 83 TAP syntax errors.
Re-run prove with the -p option to see them all.
Files=1, Tests=132,  0 wallclock secs ( 0.03 usr  0.00 sys +  0.11 cusr  0.02 csys =  0.16 CPU)
Result: FAIL
```

@atoomic @toddr 

Thank you very much.
Jim Keenan

**Comment by atoomic** [comment]
this is now fixed by 8921b2d1

**Comment by jkeenan** [comment]
LGTM
